examples/express-ex/index.js

```diff
@@ -3,8 +3,8 @@
 const express = require("express");
 const { Receiver } = require("cloudevents");
 const app = express();
-const bodyParser = require('body-parser')
-app.use(bodyParser.json())
+const bodyParser = require("body-parser");
```

Comment:
I would not expect this change in this PR.

Was this fixed with some linting tool? Perhaps we should add it as a GitHub action step to check this folder.

Can we revert this file?

Reply:
Yep